notebooks/04_model_select_and_optimize/hypertune_spark_deep_dive.ipynb

```diff
@@ -895,7 +1434,7 @@
   },
```

Comment:
Would it be possible to compare as well the model with the custom p@k loss function?

Reply:
I did not do that because p@k takes much longer time than rating metrics like rmse. Again, I am thinking to leave scalable discussions onto Azure Databricks notebooks, instead of putting everything into the current one. It is just for illustration purpose. What do you think?

Reply:
yeah, makes sense